server/client.go

```diff
@@ -101,7 +108,6 @@ func init() {
 func (c *client) initClient(tlsConn bool) {
```

Comment:
Note needed for immediate release IMO and we need efforts on STAN, just a placeholder for work to do eventually. ;)